package.json

```diff
@@ -35,7 +35,8 @@
   ],
   "dependencies": {
     "axios": "^0.18.0",
-    "bitcore-mnemonic": "^1.7.0",
+    "bitcore-lib": "^8.25.10",
```

Comment:
We have to update our `package-lock.json` as well, right?

Reply:
Done